src/components/Header/Header.css

```diff
@@ -0,0 +1,113 @@
+.cdc-react.header {
+    display: flex;
+    height: 60px;
+    padding: 0px 4px;
+    align-items: center;
+    gap: 12px;
+    border-bottom: 1px solid #E0E0E0;
+    background: #F0F0F0;
+}
+
+.cdc-react.header.logo {
+    align-self: stretch;
+    fill: #005EAA;
+    margin-left: 0.5rem;
+}
+
+.cdc-react.header.action-items {
+    justify-content: end;
+    flex-grow: 1;
+}
+
+.cdc-react.header.action-items button {
+    border-radius: 4px;
+    border: none;
+    display: flex;
+    padding: 5px;
+}
+
+.cdc-react.header.action-items button:hover {
+    background: #e6e6e6;
+}
+
+.cdc-react.header.action-items.user-profile:hover {
+    background: none;
+}
+
+.cdc-react.header.action-items.user-profile {
+    flex-grow: 0;
+}
+
+.cdc-react.header.action-items.user-profile.user-image {
+    align-items: center;
+    padding: 0px;
+}
+
+.cdc-react.header.action-items.user-profile.chevron-down {
+    margin-left: -12px;
+    margin-right: -4px;
+}
+
+.cdc-react.popup {
+    display: flex;
+    width: 240px;
+    flex-direction: column;
+    align-items: flex-start;
+    border-radius: 4px;
+    border: 1px solid #E0E0E0;
+    background: #FFF;
+    box-shadow: 0px 0px 12px 0px rgba(0, 0, 0, 0.10);
+}
+
+.cdc-react.popup button {
+    flex-direction: row;
+    display: flex;
+    min-width: 100%;
+    min-height: 44px;
+    align-items: center;
+    align-self: stretch;
+    background: #ffffff;
+    border: none;
+}
+
+.cdc-react.popup button:hover {
+    background: #e6e6e6;
+}
+
+.cdc-react.popup button:last-child {
+    border-top: solid 1px #ddd;
+}
+
+.cdc-react.popup button svg {
+    margin-right: 2px;
+    margin-left: 8px;
+}
+
+.cdc-react.popup-wrap {
+    position: absolute;
+    top: 68px;
+    right: 44px;
+}
+
+.cdc-react.notification-count {
+    flex-grow: 1;
+    justify-content: end;
+    margin-left: 100px !important;
+    margin-right: 5px !important;
+}
+
+.cdc-react.notification-count text {
+    display: flex;
+    stroke: #fff;
+    font-size: 10px;
+}
+
+@media (max-width: 480px) {
+    .cdc-react.action-item.notifications-list-item {
```

Comment:
Instead of handling the display of each item we want to hide on mobile, let's wrap them in a single container and toggle the display on that.  We also shouldn't need `!important` for that either.

Reply:
I was able to resolve without `!important`. Instead of wrapping them in a single container, could we have a reusable class such as `.hide-on-mobile` instead?

Reply:
I like that!